### ISSUE TITLE
External Configuration using Settings.XamlStyler file in parent directory not working

### ISSUE THREAD

**Issue by claudiobernasconi** [Xavalon/XamlStyler#129]
I am using the XamlStyler Visual Studio 2017 extension in version 3.0.

I am trying to setup a custom [**external configuration**](https://github.com/Xavalon/XamlStyler/wiki/External-Configurations) which will be used in various projects within the same repository. Because I do not want every single developer to import a specific file using the Visual Studio Settings importer I want to make use of the feature which allows an external configuration file called **Settings.XamlStyler** to be placed in a parent directory of the solution file.

```
Repository
└── Project1
        └── Source
        └── Build
└── Project2
        └── Source
        └── Build
└── Product1.sln
└── XamlStyler.Settings
```
As you can see in this directory structure, I want to place the settings file in the root folder of the repository, and I have several projects which should make use of this settings file.

Unfortunately, this does not work at all. The local settings defined in the options menu of Visual Studio will be used despite the **XamlStyler.Settings** file is present.

**File content:** I use the default content defined in the documentation as starting point. It defines a single JSON object containing all the properties.

**Note:** I do not want to include the settings file in any CSharp project. Although this seems possible according to the documentation, I cannot add a settings file to every project because I have more than 500 projects in this repository. I really want to make use of the possibility to have a single settings file for the entire repository.

Do I misunderstand the capabilities of the external configuration file or is this actually a bug? I have searched the code base and found a method called [GetConfigPathBetweenPaths ](https://github.com/Xavalon/XamlStyler/blob/f325bf4172308059b77e4d31c24343fc90414a60/XamlStyler3.Package/StylerPackage.cs) which seems to be doing what I expect. I did not dive deeper into the code base as I am new to this project. If you can confirm that my expected behavior should work, I would be open to debug the extension on my machine.

Any inputs would be highly appreciated. Thank you in advance.

**Comment by claudiobernasconi** [comment]
According to feedback from @NicoVermeir on Twitter, this issue should be related to and fixed by #75

**Comment by claudiobernasconi** [comment]
I can confirm that this issue is resolved by the changes made in version 3.1 of the XamlStyler Visual Studio extension. Well done!

**Comment by NicoVermeir** [comment]
thanks for reporting back :)
